app/src/main/java/com/ajdi/yassin/netfleak/model/SpokenLanguage.java

```diff
@@ -0,0 +1,44 @@
+package com.ajdi.yassin.netfleak.model;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Created by @author = GHARRAS
+ * On 17/10/2019
+ * Email med.gharras@gmail.com
+ */
+class SpokenLanguage {
```

Comment:
Instead of using a class for a simple data we can use it as a property.

Reply:
ok I will modify the class now

Reply:
I just saw the class and it need to be a list of spoken languages so I think it's will not be bad to have it as a class rather than having 2 list, unless we make it a MAP or something like that